### PR TITLE
rename references to `master` branch to `main`

### PR DESCRIPTION
**Pull request by zacharyburnett**

### CODE REVIEW

**Review by mdlpstsci — APPROVED** [review]

**Comment by zacharyburnett** [comment]
@mdlpstsci am I okay to merge this and rename the `master` branch? Let me know if you'd like for me to wait

**Comment by mdlpstsci** [comment]
@zacharyburnett @jhunkeler 
As I recall we decided not to rename the master branch to main until **The Reunion** branch is merged.  All the testing has been done on our end, and Joe has made updates.  I have just formally approved PR#412 for Joe.  If he feels it is ready to go with all changes needed at this point, then I presume he will merge.  Once the merge is done, I am not sure if there is any further reason to wait for the rename.

**Comment by zacharyburnett** [comment]
> @zacharyburnett @jhunkeler As I recall we decided not to rename the master branch to main until **The Reunion** branch is merged. All the testing has been done on our end, and Joe has made updates. I have just formally approved PR#412 for Joe. If he feels it is ready to go with all changes needed at this point, then I presume he will merge. Once the merge is done, I am not sure if there is any further reason to wait for the rename.

right, thanks! I keep forgetting that is still in the works. Looking forward to the big reunion!

**Comment by pllim** [comment]
https://github.com/spacetelescope/hstcal/pull/412 is merged!

**Comment by jhunkeler** [comment]
PS - There are more strings to replace now especially in the `.github` directory

**Comment by pllim** [comment]
PPS - Nothing a recursive grep cannot do.

`grep -r master .`

**Comment by zacharyburnett** [comment]
amended the commit to include more references to `master`

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # HSTCAL
 
-[![Jenkins CI](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/master/badge/icon)](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/master/)
+[![Jenkins CI](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/main/badge/icon)](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/main/)
```
Comment (pllim) [inline README.md:3]: Is ssbjenkins still a thing?

Comment (zacharyburnett) [reply README.md:3]: good catch; I'm not sure there is any badge that we can serve from behind the network, at the moment:
```suggestion
```

Comment (pllim) [reply README.md:3]: I thought one of the Chief Engineers killed it altogether.

**Comment by mdlpstsci** [comment]
Folks the DSB team is waiting for an HSTDP build.  I need to tag HSTCAL with an RC for testing.  When it is ready to go (hopefully not too long from now), please let me know.  Alternatively, please let me know what the issues are in a list summary.

**Review by pllim — APPROVED** [review]
Looks good, but you need to update the default branch ASAP before or after merge or things will break.

**Review by jhunkeler — APPROVED** [review]

**Comment by jhunkeler** [comment]
@zacharyburnett Go for it.

**Comment by mdlpstsci** [comment]
@zacharyburnett I merged for you.  I need to tag.

**Pull request merged**